### ISSUE TITLE
Password Issue

### ISSUE THREAD

**Issue by 1903908841** [0HugoHu/ELEC3662-Advanced-Calculator#1]
Hello blogger, I would like to reference your code. May I ask what the password is? The default password 1234 is incorrect. Thanks  a lot!

**Comment by 0HugoHu** [comment]
@1903908841 
Hi, after reviewing my code, I found there might be a bug: if no password is ever set, please first run the function _setPassword(YOUR_PASSWORD)_, to set an initial value. 

Happy holidays,
Hugo

**Comment by 1903908841** [comment]
OK, thank u very much, the issue has been successfully solved!

**Comment by 1903908841** [comment]
Hello blogger! It is me again hahahahah Sorry to disturb you again!
I found anthor issue that is: when I do some calculations, for instance, I enter '1+2', after I pressed 'D' (equal key) on the keypad,  the LCD is frozen, in other word, LCD stuck in the current display stage, but it does not display the final result on the LCD at second line.
And I pressed any key on the keypad cannot wake up the calculator, including delete, clear, the calculator has no reactions.
Is it a softwore problem or the hardware problem? 
Thanks a lot!
